app/views/occupation_standards/show.html.erb

```diff
@@ -16,111 +16,228 @@
+                <div class="-mt-0.5 border-t-2 border-seafoam-700 border-opacity-30 panel hidden w-full" tabindex="0"> <!-- begin content tab 2 -->
+                      <div class="flex justify-between p-4">
+                        <div>
+                            <h2 class="py-2 text-2xl font-bold">Related Instruction Content</h2>
+                            <p>Training Provider(s):</p>
+                            <div class="my-3 flex flex-col md:flex-row">
+                                <div class="mr-6 mb-4 flex md:items-center">
+                                    <div>
+                                        <img class="mr-2 h-7 w-auto rounded" src="https://flowbite.com/docs/images/examples/image-1@2x.jpg" alt="organization logo">
+                                    </div>
+                                    <div class="font-bold">Laney College</div>
+                                </div>
+                                <div class="mr-6 mb-4 flex md:items-center">
+                                    <div>
+                                        <img class="mr-2 h-7 w-auto rounded" src="https://flowbite.com/docs/images/examples/image-1@2x.jpg" alt="organization logo">
+                                    </div>
+                                    <div class="font-bold">ABC Technical School</div>
+                                </div>
+                            </div>
+                        </div>
+
+                        <div class="flex items-center">
+                            <div class="flex flex-col md:flex-row">
+                                <div class="m-1.5 w-28 rounded bg-pear-100 px-1 py-3 text-center">
+                                    <div class="text-3xl font-bold text-pear-700">600</div>
+                                    <div class="text-sm font-bold text-pear-800">RSI hours</div>
+                                </div>
+                            </div>
+                        </div>
+                      </div>
+                </div><!-- end content tab 2 -->
+
+            </div><!-- end occupation tabs -->
+
+        </div><!-- end purple border -->
+
+        <div class="similar-programs">
+            <h3 class="font-bold text-2xl text-primary-600 pt-4">Similar Programs</h3>
+            <div class="py-4 flex flex-col md:flex-row md:justify-between">
+                <div class="w-full h-full rounded py-4 px-2 mr-4 mb-4 border-2 border-seafoam-200 bg-white">
+                    <div class="text-primary-600 font-bold">
+                        Mechatronics Technician
+                    </div>
+                    <div class="text-orange-800">
+                        <span class="text-yonder-500">Hybrid</span>
+                        <span>&#x2022;</span>
+                        <span>17-3024.00</span>
+                    </div>
+                </div>
```

Comment:
@fatimaburke can you wrap these in `a` tags with just `href="#"` for now so that they could be clicked to go view that standard?

Reply:
@fatimaburke I was thinking more that each box would be clickable and could take you to view that individual standard. 

<img width="1303" alt="Screen Shot 2023-04-25 at 8 54 15 AM" src="https://user-images.githubusercontent.com/1938665/234334052-c17a1b31-3670-4efe-ac62-176e0e954e14.png">

Reply:
@littleforest Thanks for the clarity. I'll update.

Reply:
@fatimaburke thanks for the update! Looks great. For some reason now the 2nd box has the blue color for the ONET code. I cannot figure it out, the blocks all look the same? Do you know what is causing that?

<img width="1247" alt="Screen Shot 2023-04-25 at 11 05 02 AM" src="https://user-images.githubusercontent.com/1938665/234363897-1de66713-146c-4e9d-9755-41b4de1bb6f7.png">